setup.py

```diff
@@ -103,7 +103,12 @@ def build_extension(self, ext):
     cmdclass={"build_ext": CMakeBuild},
     zip_safe=False,
     packages=find_namespace_packages(include=["mqt.*"]),
-    install_requires=["qiskit-terra>=0.20.2,<0.22.0", "retworkx>=0.11.0,<0.12.0"],
+    install_requires=[
+        "qiskit-terra>=0.20.2,<0.22.0",
+        "retworkx>=0.11.0,<0.12.0",
+        "importlib_resources>=5.9; python_version < '3.10'",
+        "matplotlib>=3.5,<3.7",
```

Comment:
I might be wrong, but is this dependency here really needed? as far as I can tell you just use it to provide the respective type hints for the draw functions (which come from retworkx). If matplotlib is already a dependency of retworkx and we don't use it in any other way, than it might as well be dropped.

Reply:
Do you mean I shouldn't even import it? Because if I import it, I need the dependency. Otherwise the build fails.

Reply:
`retworkx` has a `mpl` extra dependency ;-) (and a graphviz one).
So you can just replace the `retworkx` dependency with `retworkx[all]>=0.11.0,<0.12.0` and remove the explicit mpl dependency here.

Reply:
I see. Thanks for the hint.